app/assets/javascripts/short-form/directives/checkbox-preference-item.html.slim

```diff
@@ -1,6 +1,6 @@
 div
   p.checkbox-block
-    input id="preferences-{{checkbox_pref_type}}" name="preferences" data-event="{{pref_data_event}}" ng-model="application.preferences[checkbox_pref_type]" type="checkbox" aria-labelledby="{{labelledby}}" ng-click='reset_preference_data(checkbox_pref_type)'
+    input id="preferences-{{checkbox_pref_type}}" name="preferences" data-event="{{pref_data_event}}" ng-model="application.preferences[checkbox_pref_type]" type="checkbox" aria-labelledby="{{labelledby}}" ng-click="reset_preference_data(checkbox_pref_type);refresh_member_dropdown()"
     label.checkbox-block_label for="preferences-{{checkbox_pref_type}}"
```

Comment:
yeah i was gonna go and put it within `select_liveWork` but it's really doing something else and didn't want that dependency.

I do like `onPreferenceSelect` that calls both though.. that's more semantic-- let me refactor.